services/media/service/src/domains/common/register-messaging.ts

```diff
@@ -25,6 +28,10 @@ export const registerCommonMessaging = (
       ImageServiceMultiTenantMessagingSettings.DeclareImageTypes,
       config,
     ).sendCommand(),
+    new RascalConfigBuilder(
```

Comment:
Can you also add handlers and listen for Success/Fail events? Just logging the received payload with a fitting message is enough. Failed - ERROR log level, Success - INFO log level.

Reply:
Added missing RabbitMQ handlers.